apps/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -315,13 +315,21 @@ export abstract class BaseInstallManager {
     const hookSource: string = path.join(this._rushConfiguration.commonFolder, 'git-hooks');
     const git: Git = new Git(this.rushConfiguration);
     const hookDestination: string | undefined = git.getHooksFolder();
+    const hooksSourceSameAsDestination: boolean = hookSource === hookDestination;
 
-    if (FileSystem.exists(hookSource) && hookDestination) {
+    if (hooksSourceSameAsDestination) {
+      console.log(
+        `git config core.hooksPath points to "common/git-hooks" folder, no need for separate git hooks installation`
+      );
+    }
+
+    if (FileSystem.exists(hookSource) && hookDestination && !hooksSourceSameAsDestination) {
       const allHookFilenames: string[] = FileSystem.readFolder(hookSource);
       // Ignore the ".sample" file(s) in this folder.
       const hookFilenames: string[] = allHookFilenames.filter((x) => !/\.sample$/.test(x));
       if (hookFilenames.length > 0) {
         console.log(os.EOL + colors.bold('Found files in the "common/git-hooks" folder.'));
+        console.log(colors.bold('Hooks will be installed into: ') + hookDestination);
```

Comment:
May want to make sure `hookDestination` is bolded as well.

Reply:
Oh this was intentional, to make a clearer separation between the static message and the dynamic info. Matter of taste I suppose. I can change it back?

Reply:
@octogonz thoughts? I don't mind either way (a differentiation would be helpful, I think?)